src/index.html

```diff
@@ -7,12 +7,64 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/index.css"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page">
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="image iMac"
+        class="card__img"
+      />
+
+      <div class="card__title">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="card__code">Product code: 195434</div>
+
+      <div class="card__reviews">
+        <div class="card__reviews_stars">
+          <div class="stars stars--1"></div>
+          <div class="stars stars--2"></div>
+          <div class="stars stars--3"></div>
+          <div class="stars stars--4"></div>
+          <div class="stars stars--5"></div>
+        </div>
+        <div class="card__reviews_count">Reviews: 5</div>
+      </div>
+
+      <div class="card__price">
+        <div class="card__price_title">Price:</div>
+        <div class="card__price_sum"></div>
+      </div>
+
+      <a
+        class="card__button"
+        data-qa="hover"
+      >
+        Buy
+      </a>
```

Comment:
The `<a>` tag is used as a button, but it lacks an `href` attribute. If this is intended to be a button, consider using a `<button>` element instead, or provide a valid `href` if it should link somewhere.